ui/src/app/settings/actions/config.js

```diff
@@ -0,0 +1,16 @@
+const config = {};
+
+config.fetch = () => {
+  return {
+    type: "WEBSOCKET_SEND",
```

Comment:
fyi, this is a bit weird, and will be [fixed shortly](https://github.com/canonical-web-and-design/maas-ui/issues/54).